modyn/trainer_server/internal/dataset/extra_local_eval/binary_file_wrapper.py

```diff
@@ -85,3 +85,11 @@ def get_samples_from_indices(self, indices: list) -> list[bytes]:
 
         samples = [data[(idx * self.record_size) + self.label_size : (idx + 1) * self.record_size] for idx in indices]
         return samples
+
+    def persist_sub_file(self, indices: list, sub_file_path: str) -> None:
+        with open(self.file_path, "rb") as file:
```

Comment:
Since this is not really using functionalites from the BinaryFileWrapper, let's move this into the train eval split python file. The "problem" is that this file here is not really supposed to exist within modyn, it's just for running local benchmarks, we should not rely on it. Let's move it to the criteo specific file.